pkg/util/util.go

```diff
@@ -105,12 +110,43 @@ func AddDefaultRegistryToImage(image string) string {
 		return ""
 	}
 
-	for k := range commonDockerRegistries {
-		if strings.HasPrefix(image, k) || strings.HasPrefix(image, "gcr.io") || strings.HasPrefix(image, "k8s.gcr.io") {
+	// check if image has version-tag included, add ":latest" if missing
+	if strings.Count(image, ":") < 1 {
+		image = image + ":latest"
+	}
+
+	parts := strings.Split(image, "/")
+
+	// no separate parts, always add registry-server
+	if len(parts) <= 0 {
```

Comment:
nit: We don't need this check as we are already checking if the image is empty above. This will be true only if the image is empty.